impl/proto/keytransparency_v1_service/keytransparency_v1_service.proto

```diff
@@ -60,4 +63,45 @@ service KeyTransparencyService {
       body: "entry_update"
     };
   }
+
+  // GetMutationsByEpoch returns a list of mutations paged by epoch number.
+  //
+  // Returns a list of mutations and their inclusion proofs along with the epoch
+  // signed map root.
+
+  rpc GetMutationsByEpoch(keytransparency.v1.types.GetMutationsRequest)
+      returns (keytransparency.v1.types.GetMutationsResponse) {
+    option (google.api.http) = { get: "/v1/mutations/{page_token}" };
+  }
+
+  // GetMutationsBySequence returns a list of mutations paged by sequence number.
+  //
+  // Returns a list of mutations and their inclusion proofs along with the epoch
+  // signed map root.
+
+  rpc GetMutationsBySequence(keytransparency.v1.types.GetMutationsRequest)
+      returns (keytransparency.v1.types.GetMutationsResponse) {
+    option (google.api.http) = { get: "/v1/mutations/{page_token}" };
```

Comment:
It looks like both this API and the one above have the same URL. Does this cause problems?